logging/source/RialtoLogging.cpp

```diff
@@ -42,91 +42,129 @@ static const firebolt::rialto::logging::EnvVariableParser g_envVariableParser;
 /**
  * Log handler for each component. By default will use journaldLogHandler.
  */
-static void consoleLogHandler(RIALTO_DEBUG_LEVEL level, const char *file, int line, const char *function,
+static void consoleLogHandler(RIALTO_COMPONENT component, RIALTO_DEBUG_LEVEL level, const char *file, int line, const char *function,
                               const char *message, size_t messageLen);
-static void journaldLogHandler(RIALTO_DEBUG_LEVEL level, const char *file, int line, const char *function,
+static void journaldLogHandler(RIALTO_COMPONENT component, RIALTO_DEBUG_LEVEL level, const char *file, int line, const char *function,
                                const char *message, size_t messageLen);
 static firebolt::rialto::logging::LogHandler g_logHandler[RIALTO_COMPONENT_LAST] = {};
+static std::string componentToString(RIALTO_COMPONENT component);
+static std::string levelToString(RIALTO_DEBUG_LEVEL level);
 
-/**
- * Console logging function for the library.
- */
-static void consoleLogHandler(RIALTO_DEBUG_LEVEL level, const char *file, int line, const char *function,
-                              const char *message, size_t messageLen)
-{
-    timespec ts = {0, 0};
-    clock_gettime(CLOCK_MONOTONIC, &ts);
-    struct iovec iov[5];
-    char tbuf[32];
-
-    iov[0].iov_base = tbuf;
-    iov[0].iov_len = snprintf(tbuf, sizeof(tbuf), "%.010lu.%.06lu ", ts.tv_sec, ts.tv_nsec / 1000);
-    iov[0].iov_len = std::min(iov[0].iov_len, sizeof(tbuf));
+static std::string componentToString(RIALTO_COMPONENT component)
+ {
+    switch (component)
+    {
+    case RIALTO_COMPONENT_DEFAULT:
+        return "DEF";
+        break;
+    case RIALTO_COMPONENT_CLIENT:
+        return "CLI";
+        break;
+    case RIALTO_COMPONENT_SERVER:
+        return "SRV";
+        break;
+    case RIALTO_COMPONENT_IPC:
+        return "IPC";
+        break;
+    case RIALTO_COMPONENT_SERVER_MANAGER:
+        return "SMG";
+        break;
+    case RIALTO_COMPONENT_COMMON:
+        return "COM";
+        break;
+    case RIALTO_COMPONENT_LAST:
```

Comment:
No need to convert 'component' RIALTO_COMPONENT_LAST to string. It's used only to know how many enum values we have.